ADC_Submission_Guidelines_Review.md

```diff
@@ -0,0 +1,110 @@
+
+Depending on the complexity of the data set and the quality of the initial submission, the review process can take anywhere from a few hours to several weeks. Long processing times generally occur when initial submissions have incomplete metadata and/or poorly organized files and/or the submitter is not responsive to our follow-up emails. Compliance with the guidelines detailed here should ensure quick processing times. Well organized and complete data sets can potentially be published within one business day. After the review process, each data set will be given a unique Digital Object Identifier (DOI) that will assist with attribution and discovery.	
+
+## Licensing and Data Distribution Policy
+
+All data and metadata published to the ADC will be released under either the CC-0 Public Domain Dedication or the Creative Commons Attribution 4.0 International License (CC-BY), with the potential exception of social science data that have certain sensitivities related to privacy or confidentiality. In cases where legal (e.g., contractual) or ethical (e.g., human subjects) restrictions to data sharing exist, requests to restrict data publication must be made in advance and in writing and are subject to the approval of NSF, who will ensure compliance with all federal, university, and Institutional Review Board policies on the use of restricted data.	
+
+## Alternative Submission Methods
+
+Most users submit to the ADC through the [ADC website](https://arcticdata.io/catalog/#share). However, there are several alternative tools to submit data sets to the ADC.
+
+- Users can submit data inside an R workflow using the [dataone R package](https://github.com/DataONEorg/rdataone).
+- Users can submit data inside a MATLAB workflow using the [Matlab DataONE library](https://github.com/DataONEorg/matlab-dataone).
+- In addition to our web and data tools shown above, the Arctic Data Center provides the ability to access and submit data via the [DataONE REST API](https://arcticdata.io/catalog/#api). This allows the community to use many programming languages to add data and metadata to the repository, search for and access data, and automate any process that might otherwise be highly time consuming. Most useful to groups with repetitive tasks to perform, such as submitting many data files of the same type, the REST API can be a real time saver.
+
```

Comment:
Add

## Submission Guidelines for Large Data Sets

The ADC does not have file size limitations for NSF funded projects. All data and metadata relevant to each project should be archived regardless of total file size. Non NSF funded projects may be subject to a one-time processing fee, depending on the total size. The [ADC website](https://arcticdata.io/catalog/#share) can handle the upload of multiple large files simultaneously. However, users with slow internet connections or users that experience any trouble uploading any files through the website should contact the ADC support team at support@arcticdata.io. The ADC support team has many options to upload large data sets when connection speed is limited or files are exceptionally large.